### PR TITLE
feat: Added locale for movies

### LINKED ISSUE #895: add locale to movie module
ref book module

### PR DESCRIPTION
**Pull request by SgtApone117**
Fixes #895
- Added locale for movie data generation(Only completed en-us)
- Added test cases for the same

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -5,3 +5,6 @@
 	path = externals/fmt
 	url = https://github.com/fmtlib/fmt.git
 
+[submodule "externals/faker-cxx"]
```
Comment (cieslarmichal) [inline .gitmodules:8]: Please remove

File: tests/modules/movie_test.cpp [hunk 2]
```diff
@@ -7,57 +7,94 @@
...
+}
+}
+
+class MovieTest : public TestWithParam<Locale>
 {
 public:
 };
 
-TEST_F(MovieTest, shouldGenerateGenre)
+TEST_F(MovieTest, shouldGenerateGenreLocale)
```
Comment (cieslarmichal) [inline tests/modules/movie_test.cpp:30]: TEST_P is needed when tests with param

**Review by cieslarmichal — CHANGES_REQUESTED** [review]
Please also remove cmake cache file

File: tests/modules/movie_test.cpp (hunk 2 above)
Comment (SgtApone117) [reply tests/modules/movie_test.cpp]: I have fixed it in the latest commit, can you please review it.

**Review by cieslarmichal — APPROVED** [review]

**Review by cieslarmichal — APPROVED** [review]

**Comment by cieslarmichal** [comment]
please fix build

**Comment by SgtApone117** [comment]
Fixed the build issues

**Pull request merged**